src/main/java/build/buf/protovalidate/internal/evaluator/EnumEvaluator.java

```diff
@@ -62,11 +62,8 @@ public boolean tautology() {
    */
   @Override
   public ValidationResult evaluate(Value val, boolean failFast) throws ExecutionException {
-    Descriptors.EnumValueDescriptor enumValue = val.value(Descriptors.EnumValueDescriptor.class);
-    if (enumValue == null) {
-      return ValidationResult.EMPTY;
-    }
-    if (!values.contains(enumValue.getNumber())) {
+    Integer enumValue = val.jvmValue(Integer.class);
+    if (!values.contains(enumValue)) {
```

Comment:
I think we should keep the null check from before as `Set.contains` is defined that it could throw a NPE if the underlying set doesn't support null values.

Reply:
Makes sense, I can do that. It's been a long time since wrote this - I don't remember if there was a reason I removed the null check or if I just tried it and it worked. Certainly the implementations used in this class now permit null, but it's good to safeguard.